starter-code/css/main.css

```diff
@@ -6,3 +6,156 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+    width: 100%;
+    margin: auto;
+    font-family: 'Montserrat', sans-serif;
+
+}
+
+.logo-and-nav {
+    height: 70px;
+    padding: 10px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.logo img {
+    width: 150px;
+    padding-right: 10px;
+}
+
+.navbar {
+    padding-right: 20px;
+    word-spacing: 20px;
+}
+
+
+.navbar a:link {
+    text-decoration: none;
+    color: silver;
+}
+
+.navbar a:visited {
+    text-decoration: none;
+}
+
+.navbar a:hover {
+    text-decoration: underline;
+}
+
+
+
+.backgroundimage {
+    background-image: url("../images/landing.jpg");
+    background-size: contain;
+    height: 350px;
+    text-align: center;
+    display: flex;
+    align-items: center;
+}
```

Comment:
Check the background-repeat: no-repeat; property, so the image won't be repeated